app/core/interfaces/many_to_one_typeahead/input.handlebars

```diff
@@ -34,22 +34,22 @@
 		padding: 14px;
 	}
 
-	#edit_field_{{name}} .twitter-typeahead {
```

Comment:
Why were all these removed?

Reply:
Hi, the template including this css id pattern was removed here:
https://github.com/directus/directus/commit/9886878bdc603e87c9b1a95ff70ab31249194ee0#diff-04e40dd2d250493b9e5a75c63ca035d7L19

so this css selector can not be used anymore.

Reply:
Makes sense @konradwww!